extensions/wikia/Qualaroo/scripts/Qualaroo.js

```diff
@@ -27,13 +27,18 @@
 		'contentLanguage': window.wgContentLanguage,
 		'pageType': window.wikiaPageType,
 		'isCorporatePage': (window.wikiaPageIsCorporate ? 'Yes' : 'No'),
+		// canonical vertical only: 'Games', 'Entertainment', 'Lifestyle', 'Wikia'
 		'verticalName': window.verticalName,
+		// all verticals
+		'fullVerticalName': window.fullVerticalName,
 		'visitorType': window.visitorType,
 		'isPowerUserAdmin': !!window.wikiaIsPowerUserAdmin,
 		'isPowerUserFrequent': !!window.wikiaIsPowerUserFrequent,
 		'isPowerUserLifetime': !!window.wikiaIsPowerUserLifetime,
 		'isLoggedIn': !!window.wgUserName,
-		'cpBenefitsModalShown': document.cookie.indexOf('cpBenefitsModalShown') > -1
+		'cpBenefitsModalShown': document.cookie.indexOf('cpBenefitsModalShown') > -1,
```

Comment:
What do you think about changing `cpBenefitsModalShown` field to more descriptive one?